hw_6/context_hw6.py

```diff
@@ -0,0 +1,31 @@
+
+class Colorized:
+    def __init__(self, key):
+        self.key = key
+        self.colors = {'WHITE': '\033[97m',
+                       'GRAY': '\033[90m',
+                       'RED': '\033[91m',
+                       'GREEN': '\033[92m',
+                       'YELLOW': '\033[93m',
+                       'ORANGE': '\033[94m',
+                       'BLUE': '\033[95m',
+                       'CYAN': '\033[96m'}
+
+    def __enter__(self):
+        print(self.colors[self.key])
+        return self.key
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        Exception(exc_type)
+        self.key = 'WHITE'
+        print(self.colors[self.key], f'Printed in default colors {self.key}')
```

Comment:
no needs to print in default color very run, remove

Reply:
corrected